packages/core/telemetry/admin/index.js

```diff
@@ -0,0 +1,24 @@
+import { ampli } from './src/ampli/index.js'; // eslint-disable-line
```

Comment:
you don't need to specify index.js in the from.

Why do we need eslint disable here ?

Reply:
For some reason, if I don't add index.js to this import, it doesn't work. When I do though, eslint screams at me that I should not add the extension.